src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesDeclarativeAgentTest.java

```diff
@@ -152,4 +153,18 @@ public void declarativeWithNestedExplicitInheritance() throws Exception {
         r.assertLogContains("Apache Maven 3.3.9", b);
         r.assertLogNotContains("go version go1.6.3", b);
     }
+
+    @Issue("JENKINS-60886")
+    @Test
+    public void declarativeWithPodRetention() throws Exception {
```

Comment:
Could you merge this test with `declarative` ? Each new Jenkins-based test adds a time overhead.